src/index.js

```diff
@@ -1,37 +1,55 @@
 import './styles/main.css';
-import Scores from './modules/score.js';
```

Comment:
[Optional] Kindly consider making your getting started section more descriptive. Add steps to guide a visitor to easily use your project. This will make your work more professional.

![image](https://user-images.githubusercontent.com/94551484/223813186-9e494272-f4f2-4521-b23b-90811fd751f9.png)